README.md

```diff
@@ -132,6 +133,7 @@ Nine strategies
 7. `RemovePodsViolatingTopologySpreadConstraint`
 8. `RemovePodsHavingTooManyRestarts`
 9. `PodLifeTime`  
+10. `RemoveFailedPods`
```

Comment:
why do we have 2 lists 🤷

Reply:
Yea, that's a good point. the ToC gets auto-generated from `./hack/update-toc.sh`. This list here doesn't bring any value.. just maintenance overhead

Reply:
I agree, once we added the ToC this list became redundant especially as we add more and more strategies. I'm fine with removing it if you want to

Reply:
best in it's own PR
also FYI github has a TOC builtin
<img width="500" alt="Screen Shot 2021-07-30 at 10 19 49 AM" src="https://user-images.githubusercontent.com/11367/127689175-3a537ff7-7331-4f35-90eb-a9ea13de636c.png">

Reply:
#611